### PR TITLE
add libnvidia-container and lsb-release-minimal

### PR DESCRIPTION
**Pull request by Dentrax**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: libnvidia-container.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/libnvidia-container
+      tag: v${{package.version}}
+      expected-commit: 6c8f1df7fd32cea3280cf2a2c6e931c9b3132465
+
+  # Fix prefix to /usr instead of /usr/local
+  - runs: sed -i 's|export prefix *= */usr/local|export prefix = /usr|' Makefile
```
Comment (rawlingsj) [inline libnvidia-container.yaml:45]: just checking, does `- uses: autoconf/configure` work here instead of using `sed`?

Comment (Dentrax) [reply libnvidia-container.yaml:45]: Unfortunately, repo doesn't provide a `./configure` file in root dir.

File: libnvidia-container.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      runs: |
+        ldd /usr/lib/libnvidia-container.so
+        ldd /usr/lib/libnvidia-container-go.so
+    - name: Test nvidia-container-cli
+      runs: nvidia-container-cli --version
+    - name: Test nvidia-container-cli with privileged mode
+      runs: |
+        set +e
+        useradd nvidia
+        # This is the expected error since `libnvidia-ml.so` must be available on the host system. Library can be found in the CUDA.
```
Comment (jamonation) [inline libnvidia-container.yaml:82]: Could the package that provides this be added? I think there's value in an affirmative test that can prove this package is able to load its dependencies.

Comment (Dentrax) [reply libnvidia-container.yaml:82]: We're going to handle this case in the image. 👍

**Review by jamonation — APPROVED** [review]

**Pull request merged**